lib/broccoli/translation-reducer/linter/find-missing-icu-arguments.js

```diff
@@ -1,6 +1,7 @@
 function findMissingIcuArguments(key, allIcuArguments, locales, icuArguments) {
   const keyIcuArguments = Array.from(allIcuArguments[key]);
-  return locales
+
+  const result = locales
```

Comment:
This could just be

```js
return locales
	.map((locale) => [
		locale,
		keyIcuArguments.filter((arg) => 
			!Object.prototype.hasOwnProperty.call(icuArguments[locale], key) || 
			!icuArguments[locale][key].includes(arg)
      	),
    ])
    .filter(([, missing]) => missing.length > 0);
```

Reply:
Makes future debugging slightly easier, going to leave this as-is